docs/source/user_guide/runtime-conf.md

```diff
@@ -273,9 +273,6 @@ Example: `test-bucket`
 and grant that key at least [Writer](https://cloud.ibm.com/docs/services/cloud-object-storage/iam?topic=cloud-object-storage-iam-bucket-permissions) level privileges.
 Specify `access_key_id` and `secret_access_key` as `cos_username` and `cos_password`, respectively.
 
-#### Kubernetes settings
-This section defines configurations that exist on the Kubernetes cluster that you want to associate with this runtime configuration.
-
 ##### Cloud Object Storage Credentials Secret (cos_secret)
```

Comment:
Can you please move this section after _Cloud Object Storage Endpoint_ and before _Cloud Object Storage Username_ so that the sequence matches the UI order? Right now a reader would have to jump back and forth to locate the information, which is disruptive.